chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/CertifiedProductManager.java

```diff
@@ -355,10 +356,10 @@ public CertifiedProductDTO changeOwnership(Long certifiedProductId, Long acbId)
     })
     @CacheEvict(value = {
             CacheNames.ALL_DEVELOPERS, CacheNames.ALL_DEVELOPERS_INCLUDING_DELETED,
-            CacheNames.COLLECTIONS_LISTINGS, CacheNames.COLLECTIONS_SEARCH,
```

Comment:
Do we still need the COLLECTIONS_LISTINGS here? I thought you would only remove COLLECTIONS_SEARCH